app/controllers/v2/content_items_controller.rb

```diff
@@ -18,17 +18,30 @@ def index
     def linkables
       # Base path is returned to facilitate rummager indexing.
       # This can be removed once link updates are picked up by rummager from the message bus.
-      render json: Queries::GetContentCollection.new(
-        document_type: query_params.fetch(:document_type),
-        fields: %w(
-          title
-          content_id
-          publication_state
-          base_path
-          internal_name
-        ),
-        pagination: NullPagination.new
-      ).call
+      document_type = query_params.fetch(:document_type)
+
+      last_update_to_document_type = ContentItem
+        .where(document_type: [document_type, "placeholder_#{document_type}"])
+        .order('updated_at ASC')
+        .select("updated_at")
+        .last
+        .updated_at
```

Comment:
Could '.pluck(:updated_at)' instead to save AR instantiation. This could also be used with proper http caching semantics and 'Rack::Cache' I think? Staleness would be provided by the updated_at.